src/SpectatorView.Unity/Assets/SpectatorView/Scripts/StateSynchronization/StateSynchronizationBroadcaster.cs

```diff
@@ -165,19 +165,17 @@ private void Update()
         public void OnFrameCompleted()
         {
             //Camera update
+            using (MemoryStream memoryStream = new MemoryStream())
+            using (BinaryWriter message = new BinaryWriter(memoryStream))
             {
-                using (MemoryStream memoryStream = new MemoryStream())
-                using (BinaryWriter message = new BinaryWriter(memoryStream))
-                {
-                    message.Write(StateSynchronizationObserver.CameraCommand);
-                    Transform camTrans = Camera.main.transform;
-                    message.Write(Time.time);
-                    message.Write(camTrans.position);
-                    message.Write(camTrans.rotation);
-                    message.Flush();
-
-                    connectionManager.Broadcast(memoryStream.ToArray());
-                }
+                message.Write(StateSynchronizationObserver.CameraCommand);
+                Transform camTrans = Camera.main?.transform ?? null;

```

Comment:
>Camera.main?.transform ?? null [](start = 37, length = 30)

?. and ?? don't work as expected with UnityEngine.Object subclasses, you need to actually do an "== null" comparison to trigger the overloaded equality operator for destroyed objects.